### DISCUSSION TITLE
Layout fields disappeared after update to v9.7.0

### DISCUSSION THREAD

**Discussion by NecoHorne** [directus/directus#12317] [Archive]
Hi

Just asking here before I open an issue. on the collection layout on the app the collection fields have disappeared after updating to v9.7.0. I don't see anything in the release notes, was it moved somewhere else? 

Example: 

v9.6.0 and previous:
![image](https://user-images.githubusercontent.com/30078491/159667454-7a8e9782-d68e-4883-a3d3-5930eab706dd.png)

v9.7.0
![image](https://user-images.githubusercontent.com/30078491/159667615-4af607fc-5928-4f25-91dd-9f1b68e8f21d.png)

I don't see any errors in the web console or application logs.

**Comment by azrikahar** [comment]
It was moved to the plus icon (also visible in your `v9.7.0` screenshot) thanks to #12154. It is in the release notes under `New Features` -> `App`, but understandably not super prominent so to speak. With that said, it was also highlighted in the announcement in our community Discord server 👍

Additional screenshot for clarity:

![image](https://user-images.githubusercontent.com/42867097/159669034-06fa56ff-13e0-4775-b079-211e0f8b0df0.png)

**Reply by NecoHorne** [reply]
Awesome thanks for the clarity. I will join the community Discord too!

**Reply by Pachat** [reply]
Be aware that it can be hidden on the rigth, under the scroll bar or the right side menu
![manual-append-cell](https://user-images.githubusercontent.com/13746133/159684850-b190d3c2-8e7f-461f-8a68-5a611bab318d.png)

**Reply by tylerforesthauser** [reply]
> Be aware that it can be hidden on the rigth, under the scroll bar or the right side menu
> 

This got me too.

**Reply by benhaynes** [reply]
Perhaps we could get some "sticky" CSS in there so that it never gets hidden under the sidebar/scrollbar? Might get messy with overlaps, but I wouldn't want this to be completely invisible. That said, _GitHub Projects_ has theirs sliding to be invisible on scroll... so maybe it's just a learning curve.

**Reply by benhaynes** [reply]
One thing that would help is when page content extends under another element (eg: sidebar) that a shadow appears to indicate that. This is the same idea as the header shadow... which only appears when the page scrolls underneath.

**Reply by Pachat** [reply]
@benhaynes What was the idea behind this `+` introduction?
It was easier to use in 9.6.0 and more logical in the sense of gathering the Layout in one place.

**Reply by rijkvanzanten** [reply]
> @benhaynes What was the idea behind this `+` introduction?
> It was easier to use in 9.6.0 and more logical in the sense of gathering the Layout in one place.

I disagree. Most users (especially non-tech) find the layout options way too easy to miss, and in general respond better to things being generally wysiwyg. It's the same with things like the calendar layout (where the date/month selection is inline), the map layout (where you can drag and drop the map), and the table layout from before (where search/column resize was already inline). If anything, this makes the options more consistent, as all important table configuration is now done on the actual table

**Reply by Pachat** [reply]
@rijkvanzanten  After all, it's a question of learning curve. As long as the users will have the `+` always visible, it's OK.
